packages/agents/router/src/config.ts

```diff
@@ -82,6 +83,7 @@ export const NxtpRouterConfigSchema = Type.Object({
   mode: TModeConfig,
   network: Type.Union([Type.Literal("testnet"), Type.Literal("mainnet"), Type.Literal("local")]),
   polling: TPollingConfig,
+  auctionRoundDepth: Type.Number(),
```

Comment:
couple things:
1. nit: may want to consider renaming this to `maxPathLength` or `maxRoutersPerTransfer` for consistency
2. we should be consulting the on-chain value, `maxRoutersPerTransfer`, to cap whatever this value can be configured to be... or perhaps we should just set this value internally *to* `maxRoutersPerTransfer` by default. At the very least, before closing out this PR please leave a TODO with this information

Reply:
If its already configured on onchain, we should be able to read the on-chain value. correct.

Reply:
UPDATE: I was wrong! This should be changed back to `auctionRoundDepth` or `maxAuctionRounds` or something. Clearly, auction rounds != maximum path length! They scale separately.